Zend/zend_language_parser.y

```diff
@@ -1295,6 +1300,10 @@ new_variable:
 			{ $$ = zend_ast_create_ex(ZEND_AST_DIM, ZEND_DIM_ALTERNATIVE_SYNTAX, $1, $3); }
 	|	new_variable T_OBJECT_OPERATOR property_name
 			{ $$ = zend_ast_create(ZEND_AST_PROP, $1, $3); }
+	|	new_variable T_NULLSAFE_OBJECT_OPERATOR property_name {
+			/* New object can never be null, we support this for consistency */
+			$$ = zend_ast_create(ZEND_AST_PROP, $1, $3);
+		}
```

Comment:
This is not what `new_variable` does. This is about cases like `new $x?->y`, which is `new ($x?->y)` and should emit a normal nullsafe access.

Reply:
Of course, now sure what got me the impression.